src/azul/__init__.py

```diff
@@ -1335,6 +1336,23 @@ def security_contact(self) -> Optional[dict[str]]:
             import json
             return json.loads(value)
 
+    @attr.s(frozen=True, kw_only=True, auto_attribs=True)
+    class SlackIntegration(TypedDict):
```

Comment:
I think we can use either `TypedDict` or `attrs` here, but I don't see any advantage to using both. I prefer `attrs` because it will raise an exception if the proper fields are not provided.

```suggestion
    class SlackIntegration:
```